tests/test_amclient.py

```diff
@@ -436,6 +469,243 @@ def test_close_completed_ingests_no_ingests(self):
         assert isinstance(close_succeeded, list)
         assert len(close_succeeded) == 0
 
+    @vcr.use_cassette('fixtures/vcr_cassettes/pipeline.yaml')
+    def test_get_pipelines(self):
+        """Test getting the pipelines available to the storage service where
+        there is at least one pipeline available to the service.
+        """
+        response = amclient.AMClient(ss_api_key=SS_API_KEY,
+                                     ss_user_name=SS_USER_NAME,
+                                     ss_url=SS_URL).get_pipelines()
+
+        objects = response['objects']
+        pipelines = objects[0]['uuid']
+        resource_uri = objects[0]['resource_uri']
+        assert amclient.is_uuid(pipelines)
+        assert resource_uri == ('/api/v2/pipeline/f914af05-c7d2'
+                                '-4611-b2eb-61cd3426d9d2/')
```

Comment:
Why are you using parenthesis here?

Reply:
@sevein I will remove in favour of backslash syntax. Parenthesis just looked better.